pmdarima/model_selection/_validation.py

```diff
@@ -115,22 +116,22 @@ def _fit_and_score(fold, estimator, y, exog, scorer, train, test, verbose,
     return test_scores, fit_time, score_time
 
 
-def _fit_and_predict(fold, estimator, y, exog, train, test, verbose):
+def _fit_and_predict(fold, estimator, y, X, train, test, verbose):
     """Fit estimator and compute scores for a given dataset split."""
     msg = 'fold=%i' % fold
     if verbose > 1:
         print("[CV] %s %s" % (msg, (64 - len(msg)) * '.'))
 
     start_time = time.time()
-    y_train, _, exog_train, exog_test = _safe_split(y, exog, train, test)
+    y_train, _, exog_train, exog_test = _safe_split(y, X, train, test)
```

Comment:
Should we rename these, or is that too much hassle?

Reply:
Yeah I can rename. Since they're just variables and not public kwargs I didn't bother, but would be nice to be consistent.